checks.d/mongo.py

```diff
@@ -200,7 +201,10 @@ def check(self, instance):
         username = parsed.get('username')
         password = parsed.get('password')
         db_name = parsed.get('database')
-        clean_server_name = server.replace(password, "*" * 5) if password is not None else server
+        clean_server_name = server.replace(
+            password,
+            "*" *
+            5) if password is not None else server
```

Comment:
the previous version is more readable. As a general rule, as long as the line is fully visible on GitHub you can leave it uncut.